webapi/src/test/scala/org/knora/webapi/e2e/v2/ListsRouteV2E2ESpec.scala

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright © 2015-2018 the contributors (see Contributors.md).
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.e2e.v2
+
+import java.net.URLEncoder
+
+import akka.actor.{ActorSystem, Props}
+import akka.http.javadsl.model.StatusCodes
+import akka.http.scaladsl.testkit.RouteTestTimeout
+import akka.pattern._
+import akka.util.Timeout
+import org.knora.webapi.messages.store.triplestoremessages.{RdfDataObject, ResetTriplestoreContent}
+import org.knora.webapi.messages.v2.responder.ontologymessages.LoadOntologiesRequestV2
+import org.knora.webapi.responders.{RESPONDER_MANAGER_ACTOR_NAME, ResponderManager}
+import org.knora.webapi.routing.v2.ListsRouteV2
+import org.knora.webapi.store.{STORE_MANAGER_ACTOR_NAME, StoreManager}
+import org.knora.webapi.{KnoraSystemInstances, LiveActorMaker, R2RSpec, SharedTestDataADM}
+import org.knora.webapi.util.{AkkaHttpUtils, FileUtil}
+import java.io.File
+
+import spray.json.{JsObject, JsValue, JsonParser}
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, ExecutionContextExecutor}
+
+/**
+  * End-to-end test specification for the search endpoint. This specification uses the Spray Testkit as documented
+  * here: http://spray.io/documentation/1.2.2/spray-testkit/
+  */
```

Comment:
This class comment isn't correct.

Reply:
https://github.com/dhlab-basel/Knora/pull/852/commits/8768e10dd53ae241ae5049df8893337ba4830a2f